src/services/telegram/app.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import cherrypy
+
+from common.WIOTRestApp import *
+from common.SettingsManager import *
+from common.SettingsNode import *
+from common.RESTBase import RESTBase
+from common.CatalogRequest import CatalogRequest
+from TelMan import MyBot
+
+class TelegramAdaptorAPI(RESTBase):
+
+    def __init__(self, upperRESTSrvcApp, settings: SettingsNode, telegramkey) -> None:
+        super().__init__(upperRESTSrvcApp, 0)
+        self._catreq = CatalogRequest(self.logger, settings)
+        self._bot = MyBot(telegramkey)
+
+    @cherrypy.tools.json_out()
+    def GET(self, *path, **args):
+
+        if path[0] == "sendMessage":
+            print("branch taken")
+            try:
+                a = self._bot.bot.sendMessage(self._bot.chat_ID,args["text"])
+            except:
+                print("An error occours")
+                return None
+
+        
+
+        # r = self._catreq.reqREST("openweatheradaptor", "/currentweather?as=2")
+
+        return a
```

Comment:
```suggestion
            return self.asjson({"something": related_to_a}) #inside the if statement
            
        # outside the if statement
        cherrypy.response.status = 404
        return self.asjson_error("Invalid request")
```

- I think it's better to not return directly `a` but something useful (if any) that doesn't not include any detailed information (maybe sensitive) 
- Edge case when the path is not recognized must return `error 404`

Reply:
Yes, agree. I would like to return just an hacknowledge.

Reply:
I decided to return True instead of a